tests/settings.py

```diff
@@ -93,3 +93,5 @@
     'oauth2_provider.oidc.handlers.EmailHandler',
     'oauth2_provider.tests.handlers.TestHandler'
 )
+
+OAUTH_RESPONSE_TYPE_CHOICES = ('code', 'token', 'id_token')
```

Comment:
Is this change still needed, given https://github.com/edx/edx-oauth2-provider/pull/34/files#diff-1da467622f70e7cdcaee448b7cd6c981?